resources/xlf/LocProject.json

```diff
@@ -90,4 +90,4 @@
       ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
@Charles-Gagnon I copied content from main branch and saved it, for some reason it adds new line at the end when I save it. Not able to find an alternative, do you know what I can do here?

Reply:
Can you not delete the newline?

Reply:
Anyways if you can't get it working for some reason it's fine to leave this one I guess - trying to revert it at this point would just be more of a pain than is worth it